c/cert/src/rules/MEM30-C/DoNotAccessFreedMemory.ql

```diff
@@ -0,0 +1,63 @@
+/**
```

Comment:
oh! if this query is the same as the [useafterfree](https://github.com/github/codeql/blob/main/cpp/ql/src/Critical/UseAfterFree.ql#L16) from standard library, we probably want to cite that somehow , so that if the upstream changes we can propagate fixes or if they are running the full coding standards+out of box queries then they can figure out why this would show up 2x

maybe something like [this](https://github.com/github/codeql-coding-standards/blob/main/c/cert/src/rules/FIO32-C/DoNotPerformFileOperationsOnDevices.ql#L21).- maybe with a CodeQL lib version number...?

Reply:
Changes made.

Reply:
Changes made.